### ISSUE TITLE
Injecting new inputs to the fully connected layer

### ISSUE THREAD

**Issue by upulcranga** [fchollet/keras#7101]
I'm just wondering whether I can inject an input vector to the fully connected layer before softmax? I'm looking to do something like what these guys have done, the architecture is shown in Fig-1. I'm using Keras with Tensorflow on a 1080ti if that helps. 

Paper - http://www.cse.lehigh.edu/~huang/Multimodal_MICCAI16.pdf

Thanks a lot in advance.
